src/RoleObjectFactory.js

```diff
@@ -365,6 +361,12 @@ function createAccessibilityObjectForRole(config) {
     } else {
       parent.accessible.addChild(displayObject);
     }
+  } else {
+    parent = displayObject.accessible.parent;
+  }
+
+  if (!containerIndex) {
+    containerIndex = displayObject.accessible.containerIndex;
```

Comment:
`containerIndex` is not a field on AccessibilityObject or any of its subclasses.  You will need to use `_.findIndex` on the parent's children array to find which is for this DisplayObject